web-app/src/store/modules/meta.store.js

```diff
@@ -0,0 +1,23 @@
+export default {
+  namespaced: true,
+
+  state: {
+    meta: null,
+  },
+
+  getters: {
+    meta: state => state.meta,
+  },
+
+  mutations: {
+    SET_META: (state, payload) => {
+      state.meta = Object.freeze(payload);
```

Comment:
Why do we need `Object.freeze` here?

Reply:
Well basically I'm just dumping data here so it can be shared between views. It doesn't need to be reactive, so freezing it gives a tiny performance benefit. (I just like to freeze all things that don't need reactivity, because it adds up, especially for large nested objects.)